omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -41,6 +41,9 @@
 
 default['private_chef']['fips_enabled'] = ChefConfig.fips?
 
+default['private_chef']['required_recipe']['enable'] = false
```

Comment:
Yeah, I can see why having two might not be necessary. I just wonder if having an on/off switch makes more usability sense than just having a path.